jobs/healing/pipeline-healing.js

```diff
@@ -229,23 +232,91 @@ async function getScopedSourceTriples( config, property, conceptSchemeUri, publi
   return reformatQueryResult(result);
 }
 
+function arrayToFile(array, file){
+  let fd = fs.openSync(file, "w");
+  for (let i=0; i<array.length; ++i){
+    fs.writeSync(fd, JSON.stringify(array[i]) + "\n")
+  }
+  fs.close(fd)
+  return file
+}
+function getTempFile() {
+  let file = "/tmp/healing-" + uuid();
+  return file;
+}
+function lines(filename) {
+  let retval = []
+  let rl = new readlines(filename)
+  let line
+  while ((line = rl.next())) {
+    line = line.toString()
+    if (line[0] === "<" || line[0] === ">") line = line.replace(/^./, "");
+    retval.push(JSON.parse(line) || {})
+  }
+  return retval;
+}
+function diffFiles(targetFile, sourceFile, S="50%", T="/tmp"){
```

Comment:
Are we sure new lines are properly escaped always? (if not, thew following example might break it right?)
```
<http://foo>	<http://bar>	""" foo 
bar """ .
```

Reply:
since the files are generated using the `arrayToFile` function and that function uses `JSON.stringify` newlines are properly escaped, e.g. your example becomes `"<http://foo>\t<http://bar>\t\"\"\" foo\nbar \"\"\" ."`